libenkf/src/enkf_main.c

```diff
@@ -2166,6 +2166,11 @@ static void enkf_main_init_obs(
   }
 }
 
+static void enkf_main_init_ensemble(enkf_main_type * enkf_main) {
```

Comment:
This is a *very limited* initialization - I fear the name creates confusion. What about: `enkf_main_add_ensemble_members( )`?